elasticsearch/src/main/java/org/hibernate/search/elasticsearch/impl/ElasticsearchIndexWorkVisitor.java

```diff
@@ -77,15 +77,19 @@ public ElasticsearchIndexWorkVisitor(String indexName, boolean refreshAfterWrite
 		this.searchIntegrator = searchIntegrator;
 	}
 
+	static String getDocumentId(LuceneWork work) {
```

Comment:
Is it used anywhere else? If not, let's make it private (I suppose it was the purpose of dropping `DocumentIdHelper`).